src/transformers/utils/quantization_config.py

```diff
@@ -24,12 +24,18 @@
 
 from packaging import version
 
-from ..utils import is_auto_awq_available, is_torch_available, logging
+from ..utils import is_auto_awq_available, is_hqq_available, is_torch_available, logging
 
 
 if is_torch_available():
     import torch
 
+if is_hqq_available():
+    from hqq.core.quantize import BaseQuantizeConfig as HQQBaseQuantizeConfig
+
+    hqq_default_config = HQQBaseQuantizeConfig(nbits=4, group_size=64, offload_meta=False)
```

Comment:
could we do this init inside the HqqConfig init?

Reply:
`hqq_default_config` is never used, can we entirely remove this? Please don't mark a conversation as being resolved until it's really gets resolved 🙏

Reply:
Sure, this was removed https://github.com/huggingface/transformers/pull/29637/files#diff-4333c40134efe7287ccda3bdd11e266b90a62629b933bf2cd15fa39cbf23b088R33-R37